client-data/js/board.js

```diff
@@ -154,10 +154,22 @@ Tools.HTML = {
 
 Tools.list = {}; // An array of all known tools. {"toolName" : {toolObject}}
 
+Tools.setToolId = function setToolId(tool) {
+	tool._id = tool.name.split(" ").join("_").toLowerCase();
```

Comment:
It may be simpler to just comme-separate the raw list of tool names in the configuration, to avoid having to add a separate `_id` that is used only for this feature.